languages/go/query.go

```diff
@@ -128,6 +132,7 @@ func (q *Query) Next() (*map[string]interface{}, error) {
 			return nil, fmt.Errorf("unexpected query event: %v", ev)
```

Comment:
Cleanup here too? Although memory leak wouldnt be the worst issue if this was hit a lot...

Reply:
Yeah this is really like a panic case. Shouldn't ever happen.